ToggleButton/ToggleButton.js

```diff
@@ -162,6 +162,7 @@ const ToggleButtonBase = kind({
 				{...props}
 				aria-pressed={props.selected}
 				css={css}
+				minWidth={false}
```

Comment:
I don't think that the `minWidth` is always `false` because there is the one case using an icon and a text together. For example, there is the regression with an `ac` icon a `d` text.

`develop` branch
![image](https://user-images.githubusercontent.com/4239873/101893069-8fbdd900-3be7-11eb-9e4c-0106d3a03ac5.png)

The PR branch
![image](https://user-images.githubusercontent.com/4239873/101893098-9c423180-3be7-11eb-8870-7164e4658eec.png)

Reply:
yes, but the the toggleButton in the develop branch was altered by the introduction of minWidth functionality in Button

Reply:
Oh I see. Let's go ahead. Thanks for letting me know.

LGTM.